design_linked_list.py

```diff
@@ -0,0 +1,73 @@
+class Node:
+    def __init__(self, val = 0, next = None):
+        self.val = val
+        self.next = next
+        
+class MyLinkedList:
+
+    def __init__(self):
+        self.head = Node()
+        self.length = 0
+
```

Comment:
Holding the current length of the linked list is a very good way than traversing everytime to check the length of the linked list. Nice Job here.